src/Sarif/Core/SarifLog.cs

```diff
@@ -101,8 +101,20 @@ public static SarifLog Load(Stream source, bool deferred = false)
                 return false;
             }
 
-            await Post(postUri, new MemoryStream(fileBytes), httpClient);
-            Console.WriteLine($"Posted log file successfully to: {postUri}");
+            HttpResponseMessage response = await Post(postUri, new MemoryStream(fileBytes), httpClient);
+            string responseText = response.Content.ReadAsStringAsync().Result;
+
+            try
+            {
+                response.EnsureSuccessStatusCode();
+            }
+            catch (Exception)
+            {
+                Console.WriteLine($"Error Posting log file to: {postUri}. Endpoint provided message: {responseText}");
+                return false;
+            }
+
+            Console.WriteLine($"Posted log file successfully to: {postUri}. Endpoint provided message: {responseText}");
             return true;
```

Comment:
I'm not sure what the format of the responseText is going to be and what it contains.  Is it possible to run this locally and provide a sample?

Reply:
This is generic, 
the server that takes the SARIF file is to be set by the user,
and that server could be with any design by the user. 
It can returns anything, e.g. correlation id.